Sources/SwiftDocC/Indexing/IndexJSON/Index.swift

```diff
@@ -0,0 +1,282 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2022 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See https://swift.org/LICENSE.txt for license information
+ See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import SymbolKit
+
+/// A navigation index of the content in a DocC archive.
+///
+/// The structure of this data is determined by the topic groups authored in
+/// the processed documentation content.
+///
+/// This is an alternative representation of the data that is also handled by the
+/// ``NavigatorIndex`` in a ``NavigatorTree``. This index is specifically designed to be emitted
+/// to disk as JSON file and implements the Index JSON spec.
+///
+/// An OpenAPI specification for Index is available in the repo at
+/// `Sources/SwiftDocC/SwiftDocC.docc/Resources/Index.spec.json`.
+public struct Index: Codable, Equatable {
```

Comment:
This is too vague IMO. `RenderIndex`?

Reply:
Agreed! I couldn't think of anything better though. `RenderIndex` makes a lot of sense to me. Thank!

Updated here: 6075a86.